app/services/wallets/validate_service.rb

```diff
@@ -48,11 +48,7 @@ def valid_granted_credits_amount?
 
     def valid_expiration_at?
       return true if args[:expiration_at].blank?
-
-      if Utils::DatetimeService.valid_format?(args[:expiration_at]) && expiration_at.to_date > Time.current.to_date
-
-        return true
-      end
+      return true if (Utils::Datetime.valid_format?(args[:expiration_at]) && expiration_at.to_date > Time.current.to_date)
```

Comment:
Style/ParenthesesAroundCondition: Don't use parentheses around the condition of an `if`. (https://rubystyle.guide#no-parens-around-condition)